Grinnell-Events-iOS/Grinnell-Events-iOS/ViewController.m

```diff
@@ -166,41 +98,17 @@ - (void)dayPicker:(MZDayPicker *)dayPicker didSelectDay:(MZDay *)day
     }
 }
 
+
 #pragma mark - UITableView Delegate Methods
 
-#pragma mark - TableView Delegate
+
+#pragma mark - TableView Delegate Methods
```

Comment:
Redundant definition: `TableView Delegate Methods` and `UITableView Delegate Methods`